_posts/2021-04-19-Time Network Of Letters.md

```diff
@@ -0,0 +1,87 @@
+
+All of these network pictures show a lot of data, so they are naturally a bit cluttered and it is hard to pick out individual connections. However, this does not mean they do not show qualitatively significant network features. Two such features that are worth looking at are:
+* The connections or lack thereof between components of the network, which can be observed from the broad view for each time period's network.
+* The connections or lack thereof between nodes (in particular those that are connected to a 'hub' node with a large number of adjacent edges) with few adjacent edges, which can be observed from the closer view for each time period's network.
+
+Broad view of letter network including letters from 1500-1650
+![1500-1650 Full Network](https://github.com/leviBlinder/HUM331-Class-Repository/blob/master/images/CytoScape1500-1650Full.png)
+
+Close up view of letter network including letters from 1500-1650
+![1500-1650 Close Network](https://github.com/leviBlinder/HUM331-Class-Repository/blob/master/images/CytoScape1500-1650Close.png)
+
+
+
+
+
+Broad view of letter network including letters from 1650-1800
+![1650-1800 Full Network](https://github.com/leviBlinder/HUM331-Class-Repository/blob/master/images/CytoScape1650-1800Full.png)
+
+Close up view of letter network including letters from 1650-1800
+![1650-1800 Close Network](https://github.com/leviBlinder/HUM331-Class-Repository/blob/master/images/CytoScape1650-1800Close.png)
+
+
+
+
+Broad view of letter network including letters from 1800-2000
+![1800-2000 Full Network](https://github.com/leviBlinder/HUM331-Class-Repository/blob/master/images/CytoScape1800-2000Full.png)
+
+Close up view of letter network including letters from 1800-2000
+![1800-2000 Close Network](https://github.com/leviBlinder/HUM331-Class-Repository/blob/master/images/CytoScape1800-2000Close.png)
+
+
+
+### How did you go about formulating a research question?
+
+The first step for formulating a research question for me was to find data suitable to network analysis. So since we read about the republic of letters and discussed it in class, this network of letter sending/receiving stood out to me as a good candidate for research on using Cytoscape. Specifically, the site we looked at in class kept metadata on the origin and destination of the letters such that each letter could be easily translated into connections between location nodes in a social network.
+
+Unfortunately, I couldn't figure out any easy way to scrape data from EMLO. Instead, I ended up using data from "correspSearch", which is an online database of the metadata of scholarly editions of letters. The site seemed to be German with metadata collected from primarily german websites/databases, which might represent a skew towards more letters with origin or destination in Germany. (However, the data from EMLO -- or any source -- would also be skewed just in a different way, since there is always selection bias when the data isn't 100% complete).
```

Comment:
This is really ambitious! I haven't poked around the EMLO site enough to see how they are navigating the problem of making their metadata accessible. It's *really* important that they do so, so I imagine they've thought about the problem in some way, but whereas other sites we've looked at (like the Mapping Early Modern London project, for example, or the Making and Knowing Project) put links to their metadata directly within their user interface, it doesn't look like EMLO has done the same thing. I imagine, since this is a database of archival materials already catalogued at libraries around the world (and libraries are very careful about preserving and maintaining metadata) that perhaps the real difficulty is that EMLO is simply pulling other repositories' metadata rather than generating it themselves, which may account for why they don't have an easily scrapable API. But not one that you should have to solve for the purposes of this class!